fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -376,6 +376,16 @@ def self.json_file?(filename)
       end
     end
 
+    # splitting on comma allows us to support comma-separated lists of values
+    # from the command line, even though the ConfigItem is not defined as an
+    # Array type
+    def self.coerce_to_array_of_strings(value)
+      return if value.nil?
+
+      value = value.split(',') unless value.kind_of?(Array)
+      return value.map(&:to_s)
+    end
+
```

Comment:
I don't think we need this added here? 🤔 This is already a thing if you define `type: Array` on the config item.

Reply:
The thing here is we would like to support both: `String` and `Array` type here dynamically:

https://github.com/fastlane/fastlane/pull/16074/files#diff-d60df02a10f3258c3bcf73b71e00bd9fR7

https://github.com/fastlane/fastlane/pull/16074/files#diff-d60df02a10f3258c3bcf73b71e00bd9fR16

If you think it's better to always force users to use `Array` of `Strings` here even for single test being passed - then your whole feedback makes sense.

Reply:
Mixed types get weird with our `Fastlane.swift` interfaces which uses types pretty heavily and it makes validation a little bit weird. We are slowly moving towards getting all the options in all the actions/tools to have a defined `type: ` 😊 We have could keep this as `Array` that would be 💯

Reply:
Ok understood ! will turn all this into `Array`s

Reply:
Thank you! ❤️